aliyun/src/test/java/org/apache/iceberg/aliyun/oss/mock/AliyunOSSMockLocalStore.java

```diff
@@ -121,8 +116,9 @@ void deleteBucket(String bucketName) throws IOException {
 
     File dir = new File(root, bucket.getName());
     if (Files.walk(dir.toPath()).anyMatch(p -> p.toFile().isFile())) {
-      throw new AliyunOSSMockLocalController.OssException(
-          409, OSSErrorCode.BUCKET_NOT_EMPTY, "The bucket you tried to delete is not empty. ");
+      // throw new OSSException(409, OSSErrorCode.BUCKET_NOT_EMPTY, "The bucket you tried to delete
```

Comment:
why do we keep this commented line?

Reply:
@jbonofre ?

Reply:
My bad, I forgot to clean. I will fix that. Thanks !

Reply:
Cleaned it up :)